app/views/merchant/dashboard/index.html.erb

```diff
@@ -1,6 +1,17 @@
 <h1>Merchant Dashboard</h1>
 
 <p><%= @merchant.name %></p>
-
 <%= link_to 'Merchant Items', merchant_items_path %>
-<%= link_to 'Merchant Invoices', merchant_invoices_path %>
\ No newline at end of file
+<%= link_to 'Merchant Invoices', merchant_invoices_path %>
+
+<div id="top-5-customers">
+  <h2>Top 5 <%= @merchant.name %> Customers</h2>
+  <ol>
+    <% @merchant.top_5_customers.each do |customer| %>
+      <div id="merchant-<%= customer.id %>">
+        <li><%= customer.name %></li>
+        <li>Number of Transactions: <%= customer.count_transactions %></li>
```

Comment:
We'll want to remove the <li> from this line.  It currently displays like this: 

1. Customer 1 Name
2. Customer 1 Transactions
3. Customer 2 Name...